manifests/2.4.0/opensearch-2.4.0.yml

```diff
@@ -83,3 +83,11 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: geospatial
+    repository: https://github.com/opensearch-project/geospatial.git
+    ref: '2.x'
+    platforms:
+      - linux
```

Comment:
I think we would need to add `windows` platform for 2.4 manifest as windows distribution is planned to released in this version. @peterzhuamazon

Reply:
Yes.

Reply:
Also the groovy failure might not relate to this tho.